frontend/src/js/components/PageViewerOrFallback.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, useEffect, useState } from 'react';
+import authFetch from '../util/auth/authFetch';
+import { useParams } from 'react-router-dom';
+import Viewer from './viewer/Viewer';
+import { PageViewer } from './PageViewer/PageViewer';
+import React from 'react';
+
+type PageViewerOrFallback = {}
```

Comment:
Is this left over from something?

Reply:
ah guess we don't need it, it doesn't accept any params in the end!